txnbuild/transaction.go

```diff
@@ -794,26 +794,26 @@ func transactionFromParsedXDR(xdrEnv xdr.TransactionEnvelope) (*GenericTransacti
 	return newTx, nil
 }
 
-// TransactionParams is a container for parameters
-// which are used to construct new Transaction instances
+// TransactionParams is a container for parameters which are used to construct
+// new Transaction instances
 type TransactionParams struct {
-	SourceAccount        Account
-	IncrementSequenceNum bool
-	Operations           []Operation
-	BaseFee              int64
-	Memo                 Memo
-	Timebounds           Timebounds
+	SourceAccount           Account
+	IncrementSequenceNum    bool
+	Operations              []Operation
+	BaseFee                 int64
+	Memo                    Memo
+	Timebounds              Timebounds
+	AdditionalPreconditions Preconditions
```

Comment:
Suggested dropping the prefix. It doesn't add much value, especially since it is basically replacing Timebounds.
```suggestion
	Preconditions Preconditions
```